tests/testthat/test-arrow-dplyr-arrange.R

```diff
@@ -151,6 +185,7 @@ test_that("arrange() on logical columns", {
 })
 
 test_that("arrange() with bad inputs", {
```

Comment:
I'm a bit confused by this block of tests - we expect them to error in the original Arrow tests, but in dplyr they don't error, and they don't via if we run them on DuckDB via Substrait.